src/equations/compressible_euler_3d.jl

```diff
@@ -944,11 +944,6 @@ References:
   Lagrangian Coordinate
   [DOI: 10.1175/MWR-D-12-00129.1](https://doi.org/10.1175/mwr-d-12-00129.1)
 """
-struct FluxLMARS{SpeedOfSound}
```

Comment:
You should also move the docstring - and update it to describe that it works in 2D as well.

Reply:
Well, you moved the `struct` but left the docstring here. Thus, the docstring in this file is not attached to anything anymore.

Reply:
Ah, I thought the docstring would rather belong to the function
`@inline function (flux_lmars::FluxLMARS)(u_ll, u_rr, orientation::Integer, equations::CompressibleEulerEquations2D)`

Reply:
Ah, I didn't see that you deleted all whitespace. Yeah, that might work. You can check whether you get the docstring in the Julia REPL help mode.

Reply:
`?FluxLMARS` gives me both 2D and 3D docstrings.